docs/customosbuildinstructions.md

```diff
@@ -93,10 +89,8 @@ Here are the expected contents of each subdirectory /file
             /bin/vhd2tar
             /bin/tar2vhd
             /bin/exportSandbox
```

Comment:
Small documentation thing: can you add `/bin/remotefs` and `/bin/netnscfg` here?